src/services/twilio-api/api-browser.js

```diff
@@ -1,90 +1,62 @@
 const fs = require('fs');
-const url = require('url');
-const { doesObjectHaveProperty } = require('../javascript-utilities');
-const ResourcePathParser = require('../resource-path-parser');
+const { camelCase } = require('../naming-conventions');
+let apiSpec; // Lazy-loaded below.
```

Comment:
Why lazy-load here, don't we always need the `apiSpec`? If so, loading it up front might be best so that during operations there are no delays. Or are you preparing for cases where we can allow users to just provide a subset of the full Twilio `apiSpec` for leaner operations?

Reply:
Lazy so it's only done if needed and only happens once. May optimize later to not need it all the time.